libraries/forms.py

```
@@ -441,7 +439,7 @@ def generate_context(
         # NOTE TO FUTURE DEVS: remember to account for the fact that a report
         #  configuration may not match with a real version in frequent cases where
         #  reports are generated before the release version has been created.
-        (report_before_release, prior_version, version) = determine_versions(
+        report_before_release, prior_version, version = determine_versions(
```

Ruff (0.15.15)

[warning] Unpacked variable `report_before_release` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)